packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -219,6 +219,14 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
     }
   }
 
+  public async disposeStreamView(userId?: string, options?: VideoStreamOptions | undefined): Promise<void> {
```

Comment:
Can `userId` here be named `remoteUserId`, then add a comment that by default it will dispose the local stream view if no remote is specified

Reply:
Also if I pass in the userId that belongs to the local user, will this fail?

Reply:
Thanks @allenplusplus, I will be working on this topic in following a few PRs, there are not just on place use string Id, will resolve them later in following PRs